theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,16 @@
+<%
+const query = site.queryPages;
+if (page.title !== 'Homepage') {
+const queryPage = query(({originalPathname}) => 
+    originalPathname.split('/').pop() == page.title.toLowerCase().replace(' ','-'));
+const breadcrumbNames = queryPage[0].originalPathname.split('/'); %>
+<div id = breadcrumb>
+<% for (name of breadcrumbNames) {
+    const queryPath = query(({title}) => name == title.toLowerCase().replace(' ','-'));
+    const lastBread = name !== breadcrumbNames[breadcrumbNames.length -1];
+    %>
+    <span <%- lastBread ? "class = 'breadcrumb-items'" : "" %> >
+    <a <%- lastBread ? "href = 'queryPath[0].pathname'" : "" %>><%= queryPath[0].title %></a>
```

Comment:
Interesting, I have just learned the usecase of `<a>` tag without `href` attribute in the breadcrumbs, I was thinking it either an invalid markup, or semantically incorrect, but seems like it is semantically correct [according to the Standards](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element), also from that document:

> If a site uses a consistent navigation toolbar on every page, then the link that would normally link to the page itself could be marked up using an a element:
```
<nav>
 <ul>
  <li> <a href="/">Home</a> </li>
  <li> <a href="/news">News</a> </li>
  <li> <a>Examples</a> </li>
  <li> <a href="/legal">Legal</a> </li>
 </ul>
</nav>
```

I think this is applicable to this case as well, good job!